packages/backend-modules/auth/lib/Consents.js

```diff
@@ -62,18 +58,16 @@ const lastRecordForPolicyForUser = async ({ userId, policy, pgdb }) =>
     },
   )
 
-const statusForPolicyForUser = async (args) =>
-  lastRecordForPolicyForUser(args).then(
+const statusForPolicyForUser = async ({ userId, policy, pgdb }) =>
+  lastRecordForPolicyForUser({ userId, policy, pgdb }).then(
     (record) => record && record.record === 'GRANT',
   )
 
 const requiredConsents = async ({ userId, pgdb }) => {
-  const { ENFORCE_CONSENTS = '' } = process.env
-
   if (ENFORCE_CONSENTS) {
     const consented = userId ? await consentsOfUser({ userId, pgdb }) : []
 
-    return ENFORCE_CONSENTS.split(',').filter(
+    return ENFORCE_CONSENTS.filter(
       (consent) => consented.indexOf(consent) === -1,
```

Comment:
```suggestion
      (consent) => !consented.includes(consent),
```
Maybe a bit easier to read?